src/components/OrderCard/OrderCard.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import OrderCardHeader from "components/OrderCardHeader";
+import OrderCardFulfillmentGroup from "components/OrderCardFulfillmentGroup";
+import OrderCardSummary from "components/OrderCardSummary";
+import PageLoading from "components/PageLoading";
+
+const styles = (theme) => ({
+  orderCard: {
+    border: `solid 1px ${theme.palette.reaction.black10}`,
+    borderRadius: "2px",
+    marginBottom: theme.spacing.unit * 2.5
+  },
+  orderCardHeader: {},
+  orderCardFulfillmentGroups: {},
+  orderCardSummary: {
+    borderTop: theme.palette.borders.default,
+    paddingBottom: theme.spacing.unit * 2,
+    paddingLeft: theme.spacing.unit * 2,
+    paddingRight: theme.spacing.unit * 2,
+    paddingTop: theme.spacing.unit * 2
+  }
+});
+
+@withStyles(styles, { withTheme: true })
+class OrderCard extends Component {
+  static propTypes = {
+    classes: PropTypes.object,
+    isExpanded: PropTypes.bool,
+    isLoadingOrders: PropTypes.bool,
+    order: PropTypes.shape({
+      email: PropTypes.string.isRequired,
+      fulfillmentGroups: PropTypes.arrayOf(PropTypes.object).isRequired,
+      payments: PropTypes.arrayOf(PropTypes.object),
+      referenceId: PropTypes.string.isRequired
+    })
+  };
+
+  renderFulfillmentGroups() {
+    const { order: { fulfillmentGroups } } = this.props;
+
+    return (
+      <Fragment>
+        {fulfillmentGroups.map((fulfillmentGroup, index) => (
+          <OrderCardFulfillmentGroup
+            key={`${index}`}
```

Comment:
It's always better to use a unique ID for `key` if there is one. In this case, you can use `fulfillmentGroup._id` as long as you request it in the GraphQL query.